src/snyk/snykCode/codeService.ts

```diff
@@ -18,6 +17,7 @@ import { IVSCodeWindow } from '../common/vscode/window';
 import { IVSCodeWorkspace } from '../common/vscode/workspace';
 import SnykCodeAnalyzer from './analyzer/analyzer';
 import { Progress } from './analyzer/progress';
+import { ISnykCodeErrorHandler } from './error/snykCodeErrorHandler';
```

Comment:
@attriaayush I've touched this file but then removed the code. Since we have autoformatting enabled on save - imports were reordered. See the setting - https://github.com/snyk/vscode-extension/blob/7a8830d6a3fa4ef43154ffcdcb7bec81609d4e11/.vscode/settings.json#L16. Not sure why it was in the wrong place before that though.